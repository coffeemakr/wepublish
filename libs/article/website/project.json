{
  "name": "article-website",
  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
  "sourceRoot": "libs/article/website/src",
  "projectType": "library",
  "tags": [],
  "targets": {
    "lint": {
      "executor": "@nx/linter:eslint",
      "outputs": ["{options.outputFile}"],
      "options": {
        "lintFilePatterns": ["libs/article/website/**/*.{ts,tsx,js,jsx}"]
      }
    },
    "build": {
      "executor": "@nx/js:tsc",
      "outputs": ["{options.outputPath}"],
      "options": {
        "outputPath": "dist/libs/article/website",
        "tsConfig": "libs/article/website/tsconfig.lib.json",
        "packageJson": "libs/article/website/package.json",
        "main": "libs/article/website/src/index.ts",
        "assets": ["libs/article/website/*.md"],
        "updateBuildableProjectDepsInPackageJson": true
      }
    },
    "test": {
      "executor": "@nx/jest:jest",
      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
      "options": {
        "jestConfig": "libs/article/website/jest.config.ts",
        "passWithNoTests": true
      }
    }
  }
}
