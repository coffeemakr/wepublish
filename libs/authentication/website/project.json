{
  "name": "authentication-website",
  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
  "sourceRoot": "libs/authentication/website/src",
  "projectType": "library",
  "tags": [],
  "targets": {
    "lint": {
      "executor": "@nx/linter:eslint",
      "outputs": ["{options.outputFile}"],
      "options": {
        "lintFilePatterns": ["libs/authentication/website/**/*.{ts,tsx,js,jsx}"]
      }
    },
    "build": {
      "executor": "@nx/js:tsc",
      "outputs": ["{options.outputPath}"],
      "options": {
        "outputPath": "dist/libs/authentication/website",
        "tsConfig": "libs/authentication/website/tsconfig.lib.json",
        "packageJson": "libs/authentication/website/package.json",
        "main": "libs/authentication/website/src/index.ts",
        "assets": ["libs/authentication/website/*.md"],
        "updateBuildableProjectDepsInPackageJson": true
      }
    },
    "test": {
      "executor": "@nx/jest:jest",
      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
      "options": {
        "jestConfig": "libs/authentication/website/jest.config.ts",
        "passWithNoTests": true
      }
    }
  }
}
