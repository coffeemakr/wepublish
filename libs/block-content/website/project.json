{
  "name": "block-content-website",
  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
  "sourceRoot": "libs/block-content/website/src",
  "projectType": "library",
  "tags": [],
  "targets": {
    "lint": {
      "executor": "@nx/linter:eslint",
      "outputs": ["{options.outputFile}"],
      "options": {
        "lintFilePatterns": ["libs/block-content/website/**/*.{ts,tsx,js,jsx}"]
      }
    },
    "build": {
      "executor": "@nx/js:tsc",
      "outputs": ["{options.outputPath}"],
      "options": {
        "outputPath": "dist/libs/block-content/website",
        "tsConfig": "libs/block-content/website/tsconfig.lib.json",
        "packageJson": "libs/block-content/website/package.json",
        "main": "libs/block-content/website/src/index.ts",
        "assets": ["libs/block-content/website/*.md"],
        "updateBuildableProjectDepsInPackageJson": true
      }
    },
    "test": {
      "executor": "@nx/jest:jest",
      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
      "options": {
        "jestConfig": "libs/block-content/website/jest.config.ts",
        "passWithNoTests": true
      }
    }
  }
}
