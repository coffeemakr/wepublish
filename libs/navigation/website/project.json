{
  "name": "navigation-website",
  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
  "sourceRoot": "libs/navigation/website/src",
  "projectType": "library",
  "tags": [],
  "targets": {
    "lint": {
      "executor": "@nx/linter:eslint",
      "outputs": ["{options.outputFile}"],
      "options": {
        "lintFilePatterns": ["libs/navigation/website/**/*.{ts,tsx,js,jsx}"]
      }
    },
    "build": {
      "executor": "@nx/js:tsc",
      "outputs": ["{options.outputPath}"],
      "options": {
        "outputPath": "dist/libs/navigation/website",
        "tsConfig": "libs/navigation/website/tsconfig.lib.json",
        "packageJson": "libs/navigation/website/package.json",
        "main": "libs/navigation/website/src/index.ts",
        "assets": ["libs/navigation/website/*.md"],
        "updateBuildableProjectDepsInPackageJson": true
      }
    },
    "test": {
      "executor": "@nx/jest:jest",
      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
      "options": {
        "jestConfig": "libs/navigation/website/jest.config.ts",
        "passWithNoTests": true
      }
    }
  }
}
