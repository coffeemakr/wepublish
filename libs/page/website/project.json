{
  "name": "page-website",
  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
  "sourceRoot": "libs/page/website/src",
  "projectType": "library",
  "tags": [],
  "targets": {
    "lint": {
      "executor": "@nx/linter:eslint",
      "outputs": ["{options.outputFile}"],
      "options": {
        "lintFilePatterns": ["libs/page/website/**/*.{ts,tsx,js,jsx}"]
      }
    },
    "build": {
      "executor": "@nx/js:tsc",
      "outputs": ["{options.outputPath}"],
      "options": {
        "outputPath": "dist/libs/page/website",
        "tsConfig": "libs/page/website/tsconfig.lib.json",
        "packageJson": "libs/page/website/package.json",
        "main": "libs/page/website/src/index.ts",
        "assets": ["libs/page/website/*.md"],
        "updateBuildableProjectDepsInPackageJson": true
      }
    },
    "test": {
      "executor": "@nx/jest:jest",
      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
      "options": {
        "jestConfig": "libs/page/website/jest.config.ts",
        "passWithNoTests": true
      }
    }
  }
}
