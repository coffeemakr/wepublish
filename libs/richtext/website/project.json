{
  "name": "richtext-website",
  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
  "sourceRoot": "libs/richtext/website/src",
  "projectType": "library",
  "tags": [],
  "targets": {
    "lint": {
      "executor": "@nx/linter:eslint",
      "outputs": ["{options.outputFile}"],
      "options": {
        "lintFilePatterns": ["libs/richtext/website/**/*.{ts,tsx,js,jsx}"]
      }
    },
    "build": {
      "executor": "@nx/js:tsc",
      "outputs": ["{options.outputPath}"],
      "options": {
        "outputPath": "dist/libs/richtext/website",
        "tsConfig": "libs/richtext/website/tsconfig.lib.json",
        "packageJson": "libs/richtext/website/package.json",
        "main": "libs/richtext/website/src/index.ts",
        "assets": ["libs/richtext/website/*.md"],
        "updateBuildableProjectDepsInPackageJson": true
      }
    },
    "test": {
      "executor": "@nx/jest:jest",
      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
      "options": {
        "jestConfig": "libs/richtext/website/jest.config.ts",
        "passWithNoTests": true
      }
    }
  }
}
