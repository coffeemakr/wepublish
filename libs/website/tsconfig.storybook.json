{
  "extends": "./tsconfig.json",
  "compilerOptions": {
    "emitDecoratorMetadata": true,
    "outDir": ""
  },
  "files": [
    "../../node_modules/@nx/react/typings/styled-jsx.d.ts",
    "../../node_modules/@nx/react/typings/cssmodule.d.ts",
    "../../node_modules/@nx/react/typings/image.d.ts"
  ],
  "exclude": ["src/**/*.spec.ts", "src/**/*.spec.js", "src/**/*.spec.tsx", "src/**/*.spec.jsx"],
  "include": [
    "../**/website/src/**/*.stories.ts",
    "../**/website/src/**/*.stories.js",
    "../**/website/src/**/*.stories.jsx",
    "../**/website/src/**/*.stories.tsx",
    "../**/website/src/**/*.stories.mdx",
    ".storybook/*.js",
    "preview.tsx",
    "main.ts"
  ]
}
