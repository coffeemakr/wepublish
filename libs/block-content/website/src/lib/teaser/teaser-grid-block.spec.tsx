import {runStorybookTests} from '@wepublish/testing/website'
import * as stories from './teaser-grid-block.stories'

describe('Teaser Grid Block', () => {
  runStorybookTests(stories)
})
