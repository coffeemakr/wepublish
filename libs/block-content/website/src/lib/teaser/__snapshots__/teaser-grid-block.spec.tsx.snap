// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teaser Grid Block should render OneColumn 1`] = `
<DocumentFragment>
  .emotion-0 {
  display: grid;
  -webkit-column-gap: 16px;
  column-gap: 16px;
  row-gap: 40px;
  grid-template-columns: 1fr;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
}

.emotion-2 {
  display: grid;
  grid-column: 1/-1;
}

@media (min-width:900px) {
  .emotion-2 {
    grid-column-start: 1;
    grid-column-end: 13;
    grid-row-start: 1;
    grid-row-end: 2;
  }
}

.emotion-4 {
  margin: 0;
  font: inherit;
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  display: grid;
}

.emotion-5 {
  display: grid;
  gap: 8px;
  grid-template-rows: auto;
  grid-auto-rows: max-content;
}

.emotion-7 {
  max-width: 100%;
  height: auto;
  aspect-ratio: auto 1;
  max-height: 80lvh;
  object-fit: contain;
  height: 100%;
  width: 100%;
  object-fit: cover;
}

.emotion-13 {
  margin: 0;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 400;
  font-size: 1.5rem;
  line-height: 1.334;
  letter-spacing: 0em;
}

.emotion-14 {
  margin: 0;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 500;
  font-size: 1.25rem;
  line-height: 1.6;
  letter-spacing: 0.0075em;
}

.emotion-15 {
  display: grid;
  grid-column: 1/-1;
}

@media (min-width:900px) {
  .emotion-15 {
    grid-column-start: 1;
    grid-column-end: 13;
    grid-row-start: 2;
    grid-row-end: 3;
  }
}

.emotion-26 {
  display: grid;
  grid-column: 1/-1;
}

@media (min-width:900px) {
  .emotion-26 {
    grid-column-start: 1;
    grid-column-end: 13;
    grid-row-start: 3;
    grid-row-end: 4;
  }
}

<div
    class="emotion-0 emotion-1"
  >
    <article
      class="emotion-2 emotion-3"
      x="0"
      y="0"
    >
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone emotion-4"
        href="https://example.com"
      >
        <div
          class="emotion-5 emotion-6"
        >
          <img
            alt="An image description"
            class="emotion-7 emotion-8"
            loading="lazy"
            srcset="https://unsplash.it/800/800 800w,
https://unsplash.it/500/500 500w,
https://unsplash.it/300/300 300w,
https://unsplash.it/200/200 200w"
            title="An image title"
          />
          <time
            class="emotion-9 emotion-10"
            datetime="2023-01-01T00:00:00.000Z"
          >
            01.01.2023
          </time>
          <header
            class="emotion-11 emotion-12"
          >
            <h1
              class="MuiTypography-root MuiTypography-h5 emotion-13"
            >
              Pre Title: Title
            </h1>
          </header>
          <p
            class="MuiTypography-root MuiTypography-h6 emotion-14"
          >
            Lead
          </p>
        </div>
      </a>
    </article>
    <article
      class="emotion-15 emotion-3"
      x="0"
      y="1"
    >
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone emotion-4"
        href="https://example.com"
      >
        <div
          class="emotion-5 emotion-6"
        >
          <img
            alt="An image description"
            class="emotion-7 emotion-8"
            loading="lazy"
            srcset="https://unsplash.it/800/800 800w,
https://unsplash.it/500/500 500w,
https://unsplash.it/300/300 300w,
https://unsplash.it/200/200 200w"
            title="An image title"
          />
          <header
            class="emotion-11 emotion-12"
          >
            <h1
              class="MuiTypography-root MuiTypography-h5 emotion-13"
            >
              Pre Title: Title
            </h1>
          </header>
          <p
            class="MuiTypography-root MuiTypography-h6 emotion-14"
          >
            Lead
          </p>
        </div>
      </a>
    </article>
    <article
      class="emotion-26 emotion-3"
      x="0"
      y="2"
    >
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone emotion-4"
        href="https://example.com"
      >
        <div
          class="emotion-5 emotion-6"
        >
          <img
            alt="An image description"
            class="emotion-7 emotion-8"
            loading="lazy"
            srcset="https://unsplash.it/800/800 800w,
https://unsplash.it/500/500 500w,
https://unsplash.it/300/300 300w,
https://unsplash.it/200/200 200w"
            title="An image title"
          />
          <header
            class="emotion-11 emotion-12"
          >
            <h1
              class="MuiTypography-root MuiTypography-h5 emotion-13"
            >
              Teambesprechung vom 23.05.
            </h1>
          </header>
          <p
            class="MuiTypography-root MuiTypography-h6 emotion-14"
          >
            Lead
          </p>
        </div>
      </a>
    </article>
  </div>
</DocumentFragment>
`;

exports[`Teaser Grid Block should render ThreeColumns 1`] = `
<DocumentFragment>
  .emotion-0 {
  display: grid;
  -webkit-column-gap: 16px;
  column-gap: 16px;
  row-gap: 40px;
  grid-template-columns: 1fr;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
}

@media (min-width:600px) {
  .emotion-0 {
    grid-template-columns: 1fr 1fr;
  }
}

@media (min-width:900px) {
  .emotion-0 {
    grid-template-columns: repeat(12, 1fr);
  }
}

.emotion-2 {
  display: grid;
}

@media (min-width:900px) {
  .emotion-2 {
    grid-column-start: 1;
    grid-column-end: 5;
    grid-row-start: 1;
    grid-row-end: 2;
  }
}

.emotion-4 {
  margin: 0;
  font: inherit;
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  display: grid;
}

.emotion-5 {
  display: grid;
  gap: 8px;
  grid-template-rows: auto;
  grid-auto-rows: max-content;
}

.emotion-7 {
  max-width: 100%;
  height: auto;
  aspect-ratio: auto 1;
  max-height: 80lvh;
  object-fit: contain;
  height: 100%;
  width: 100%;
  object-fit: cover;
}

.emotion-13 {
  margin: 0;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 400;
  font-size: 1.5rem;
  line-height: 1.334;
  letter-spacing: 0em;
}

.emotion-14 {
  margin: 0;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 500;
  font-size: 1.25rem;
  line-height: 1.6;
  letter-spacing: 0.0075em;
}

.emotion-15 {
  display: grid;
}

@media (min-width:900px) {
  .emotion-15 {
    grid-column-start: 5;
    grid-column-end: 9;
    grid-row-start: 1;
    grid-row-end: 2;
  }
}

.emotion-26 {
  display: grid;
}

@media (min-width:900px) {
  .emotion-26 {
    grid-column-start: 9;
    grid-column-end: 13;
    grid-row-start: 1;
    grid-row-end: 2;
  }
}

<div
    class="emotion-0 emotion-1"
  >
    <article
      class="emotion-2 emotion-3"
      x="0"
      y="0"
    >
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone emotion-4"
        href="https://example.com"
      >
        <div
          class="emotion-5 emotion-6"
        >
          <img
            alt="An image description"
            class="emotion-7 emotion-8"
            loading="lazy"
            srcset="https://unsplash.it/800/800 800w,
https://unsplash.it/500/500 500w,
https://unsplash.it/300/300 300w,
https://unsplash.it/200/200 200w"
            title="An image title"
          />
          <time
            class="emotion-9 emotion-10"
            datetime="2023-01-01T00:00:00.000Z"
          >
            01.01.2023
          </time>
          <header
            class="emotion-11 emotion-12"
          >
            <h1
              class="MuiTypography-root MuiTypography-h5 emotion-13"
            >
              Pre Title: Title
            </h1>
          </header>
          <p
            class="MuiTypography-root MuiTypography-h6 emotion-14"
          >
            Lead
          </p>
        </div>
      </a>
    </article>
    <article
      class="emotion-15 emotion-3"
      x="4"
      y="0"
    >
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone emotion-4"
        href="https://example.com"
      >
        <div
          class="emotion-5 emotion-6"
        >
          <img
            alt="An image description"
            class="emotion-7 emotion-8"
            loading="lazy"
            srcset="https://unsplash.it/800/800 800w,
https://unsplash.it/500/500 500w,
https://unsplash.it/300/300 300w,
https://unsplash.it/200/200 200w"
            title="An image title"
          />
          <header
            class="emotion-11 emotion-12"
          >
            <h1
              class="MuiTypography-root MuiTypography-h5 emotion-13"
            >
              Pre Title: Title
            </h1>
          </header>
          <p
            class="MuiTypography-root MuiTypography-h6 emotion-14"
          >
            Lead
          </p>
        </div>
      </a>
    </article>
    <article
      class="emotion-26 emotion-3"
      x="8"
      y="0"
    >
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone emotion-4"
        href="https://example.com"
      >
        <div
          class="emotion-5 emotion-6"
        >
          <img
            alt="An image description"
            class="emotion-7 emotion-8"
            loading="lazy"
            srcset="https://unsplash.it/800/800 800w,
https://unsplash.it/500/500 500w,
https://unsplash.it/300/300 300w,
https://unsplash.it/200/200 200w"
            title="An image title"
          />
          <header
            class="emotion-11 emotion-12"
          >
            <h1
              class="MuiTypography-root MuiTypography-h5 emotion-13"
            >
              Teambesprechung vom 23.05.
            </h1>
          </header>
          <p
            class="MuiTypography-root MuiTypography-h6 emotion-14"
          >
            Lead
          </p>
        </div>
      </a>
    </article>
  </div>
</DocumentFragment>
`;

exports[`Teaser Grid Block should render WithClassName 1`] = `
<DocumentFragment>
  .emotion-0 {
  display: grid;
  -webkit-column-gap: 16px;
  column-gap: 16px;
  row-gap: 40px;
  grid-template-columns: 1fr;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
}

@media (min-width:600px) {
  .emotion-0 {
    grid-template-columns: 1fr 1fr;
  }
}

@media (min-width:900px) {
  .emotion-0 {
    grid-template-columns: repeat(12, 1fr);
  }
}

.emotion-2 {
  display: grid;
}

@media (min-width:900px) {
  .emotion-2 {
    grid-column-start: 1;
    grid-column-end: 5;
    grid-row-start: 1;
    grid-row-end: 2;
  }
}

.emotion-4 {
  margin: 0;
  font: inherit;
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  display: grid;
}

.emotion-5 {
  display: grid;
  gap: 8px;
  grid-template-rows: auto;
  grid-auto-rows: max-content;
}

.emotion-7 {
  max-width: 100%;
  height: auto;
  aspect-ratio: auto 1;
  max-height: 80lvh;
  object-fit: contain;
  height: 100%;
  width: 100%;
  object-fit: cover;
}

.emotion-13 {
  margin: 0;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 400;
  font-size: 1.5rem;
  line-height: 1.334;
  letter-spacing: 0em;
}

.emotion-14 {
  margin: 0;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 500;
  font-size: 1.25rem;
  line-height: 1.6;
  letter-spacing: 0.0075em;
}

.emotion-15 {
  display: grid;
}

@media (min-width:900px) {
  .emotion-15 {
    grid-column-start: 5;
    grid-column-end: 9;
    grid-row-start: 1;
    grid-row-end: 2;
  }
}

.emotion-26 {
  display: grid;
}

@media (min-width:900px) {
  .emotion-26 {
    grid-column-start: 9;
    grid-column-end: 13;
    grid-row-start: 1;
    grid-row-end: 2;
  }
}

<div
    class="extra-classname emotion-0 emotion-1"
  >
    <article
      class="emotion-2 emotion-3"
      x="0"
      y="0"
    >
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone emotion-4"
        href="https://example.com"
      >
        <div
          class="emotion-5 emotion-6"
        >
          <img
            alt="An image description"
            class="emotion-7 emotion-8"
            loading="lazy"
            srcset="https://unsplash.it/800/800 800w,
https://unsplash.it/500/500 500w,
https://unsplash.it/300/300 300w,
https://unsplash.it/200/200 200w"
            title="An image title"
          />
          <time
            class="emotion-9 emotion-10"
            datetime="2023-01-01T00:00:00.000Z"
          >
            01.01.2023
          </time>
          <header
            class="emotion-11 emotion-12"
          >
            <h1
              class="MuiTypography-root MuiTypography-h5 emotion-13"
            >
              Pre Title: Title
            </h1>
          </header>
          <p
            class="MuiTypography-root MuiTypography-h6 emotion-14"
          >
            Lead
          </p>
        </div>
      </a>
    </article>
    <article
      class="emotion-15 emotion-3"
      x="4"
      y="0"
    >
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone emotion-4"
        href="https://example.com"
      >
        <div
          class="emotion-5 emotion-6"
        >
          <img
            alt="An image description"
            class="emotion-7 emotion-8"
            loading="lazy"
            srcset="https://unsplash.it/800/800 800w,
https://unsplash.it/500/500 500w,
https://unsplash.it/300/300 300w,
https://unsplash.it/200/200 200w"
            title="An image title"
          />
          <header
            class="emotion-11 emotion-12"
          >
            <h1
              class="MuiTypography-root MuiTypography-h5 emotion-13"
            >
              Pre Title: Title
            </h1>
          </header>
          <p
            class="MuiTypography-root MuiTypography-h6 emotion-14"
          >
            Lead
          </p>
        </div>
      </a>
    </article>
    <article
      class="emotion-26 emotion-3"
      x="8"
      y="0"
    >
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone emotion-4"
        href="https://example.com"
      >
        <div
          class="emotion-5 emotion-6"
        >
          <img
            alt="An image description"
            class="emotion-7 emotion-8"
            loading="lazy"
            srcset="https://unsplash.it/800/800 800w,
https://unsplash.it/500/500 500w,
https://unsplash.it/300/300 300w,
https://unsplash.it/200/200 200w"
            title="An image title"
          />
          <header
            class="emotion-11 emotion-12"
          >
            <h1
              class="MuiTypography-root MuiTypography-h5 emotion-13"
            >
              Teambesprechung vom 23.05.
            </h1>
          </header>
          <p
            class="MuiTypography-root MuiTypography-h6 emotion-14"
          >
            Lead
          </p>
        </div>
      </a>
    </article>
  </div>
</DocumentFragment>
`;

exports[`Teaser Grid Block should render WithEmotion 1`] = `
<DocumentFragment>
  .emotion-0 {
  display: grid;
  -webkit-column-gap: 16px;
  column-gap: 16px;
  row-gap: 40px;
  grid-template-columns: 1fr;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
}

@media (min-width:600px) {
  .emotion-0 {
    grid-template-columns: 1fr 1fr;
  }
}

@media (min-width:900px) {
  .emotion-0 {
    grid-template-columns: repeat(12, 1fr);
  }
}

.emotion-2 {
  display: grid;
}

@media (min-width:900px) {
  .emotion-2 {
    grid-column-start: 1;
    grid-column-end: 5;
    grid-row-start: 1;
    grid-row-end: 2;
  }
}

.emotion-4 {
  margin: 0;
  font: inherit;
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  display: grid;
}

.emotion-5 {
  display: grid;
  gap: 8px;
  grid-template-rows: auto;
  grid-auto-rows: max-content;
}

.emotion-7 {
  max-width: 100%;
  height: auto;
  aspect-ratio: auto 1;
  max-height: 80lvh;
  object-fit: contain;
  height: 100%;
  width: 100%;
  object-fit: cover;
}

.emotion-13 {
  margin: 0;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 400;
  font-size: 1.5rem;
  line-height: 1.334;
  letter-spacing: 0em;
}

.emotion-14 {
  margin: 0;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 500;
  font-size: 1.25rem;
  line-height: 1.6;
  letter-spacing: 0.0075em;
}

.emotion-15 {
  display: grid;
}

@media (min-width:900px) {
  .emotion-15 {
    grid-column-start: 5;
    grid-column-end: 9;
    grid-row-start: 1;
    grid-row-end: 2;
  }
}

.emotion-26 {
  display: grid;
}

@media (min-width:900px) {
  .emotion-26 {
    grid-column-start: 9;
    grid-column-end: 13;
    grid-row-start: 1;
    grid-row-end: 2;
  }
}

<div
    class="emotion-0 emotion-1"
  >
    <article
      class="emotion-2 emotion-3"
      x="0"
      y="0"
    >
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone emotion-4"
        href="https://example.com"
      >
        <div
          class="emotion-5 emotion-6"
        >
          <img
            alt="An image description"
            class="emotion-7 emotion-8"
            loading="lazy"
            srcset="https://unsplash.it/800/800 800w,
https://unsplash.it/500/500 500w,
https://unsplash.it/300/300 300w,
https://unsplash.it/200/200 200w"
            title="An image title"
          />
          <time
            class="emotion-9 emotion-10"
            datetime="2023-01-01T00:00:00.000Z"
          >
            01.01.2023
          </time>
          <header
            class="emotion-11 emotion-12"
          >
            <h1
              class="MuiTypography-root MuiTypography-h5 emotion-13"
            >
              Pre Title: Title
            </h1>
          </header>
          <p
            class="MuiTypography-root MuiTypography-h6 emotion-14"
          >
            Lead
          </p>
        </div>
      </a>
    </article>
    <article
      class="emotion-15 emotion-3"
      x="4"
      y="0"
    >
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone emotion-4"
        href="https://example.com"
      >
        <div
          class="emotion-5 emotion-6"
        >
          <img
            alt="An image description"
            class="emotion-7 emotion-8"
            loading="lazy"
            srcset="https://unsplash.it/800/800 800w,
https://unsplash.it/500/500 500w,
https://unsplash.it/300/300 300w,
https://unsplash.it/200/200 200w"
            title="An image title"
          />
          <header
            class="emotion-11 emotion-12"
          >
            <h1
              class="MuiTypography-root MuiTypography-h5 emotion-13"
            >
              Pre Title: Title
            </h1>
          </header>
          <p
            class="MuiTypography-root MuiTypography-h6 emotion-14"
          >
            Lead
          </p>
        </div>
      </a>
    </article>
    <article
      class="emotion-26 emotion-3"
      x="8"
      y="0"
    >
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone emotion-4"
        href="https://example.com"
      >
        <div
          class="emotion-5 emotion-6"
        >
          <img
            alt="An image description"
            class="emotion-7 emotion-8"
            loading="lazy"
            srcset="https://unsplash.it/800/800 800w,
https://unsplash.it/500/500 500w,
https://unsplash.it/300/300 300w,
https://unsplash.it/200/200 200w"
            title="An image title"
          />
          <header
            class="emotion-11 emotion-12"
          >
            <h1
              class="MuiTypography-root MuiTypography-h5 emotion-13"
            >
              Teambesprechung vom 23.05.
            </h1>
          </header>
          <p
            class="MuiTypography-root MuiTypography-h6 emotion-14"
          >
            Lead
          </p>
        </div>
      </a>
    </article>
  </div>
</DocumentFragment>
`;
